store/bench_test.go

```diff
@@ -0,0 +1,235 @@
+package store
```

Comment:
This could use more benchmarks, including perhaps some range queries. But this seems sufficient for now.